### PR TITLE
A few add-ons to v4.0 genomes freq_ht

### PR DESCRIPTION
**Pull request by KoalaQin**
In this PR: 
1. change the chrX test gene to a gene in X nonPAR region to make sure the AN computation is correct; 
2. set downsampling struct to all NA for new variants
3. add age distribution in frequency global
4. merge variant qual_hists and  age_hists from v3.1 release HT to the freq_ht.

### CODE REVIEW

**Comment by KoalaQin** [comment]
Back to you! I ran the test for the last step, but the whole for the last step has been running on a max-500 cluster for 2.5h now, should I just kill it and relaunch with max-1000?

File: gnomad_qc/v4/annotations/generate_freq_genomes.py [hunk 1]
```diff
@@ -975,6 +972,8 @@ def generate_v4_genomes_callstats(ht: hl.Table, an_ht: hl.Table) -> hl.Table:
     )
     ht = ht.annotate(freq=set_female_y_metrics_to_na_expr(ht))
 
+    ht = set_downsampling_freq_missing(ht, an_ht)
```
Comment (jkgoodrich) [inline gnomad_qc/v4/annotations/generate_freq_genomes.py:975]:
```suggestion
    logger.info(
        "Set downsampling call stats for new variants in gnomad v4.0 genomes to "
        "missing structs..."
    )
    ht = set_downsampling_freq_missing(ht, an_ht)
```

File: gnomad_qc/v4/annotations/generate_freq_genomes.py [hunk 2]
```diff
@@ -1008,11 +1007,105 @@ def generate_v4_genomes_callstats(ht: hl.Table, an_ht: hl.Table) -> hl.Table:
         freq_meta=freq_meta,
         faf_meta=faf_meta,
         faf_index_dict=make_freq_index_dict_from_meta(faf_meta),
```
Comment (jkgoodrich) [inline gnomad_qc/v4/annotations/generate_freq_genomes.py:1009]: Add this
```
faf_index_dict = make_freq_index_dict_from_meta(faf_meta)
```
above the comment "# Change the 'pop' keys in the freq_meta and faf_meta arrays to 'gen_anc'."

And change this line to `faf_index_dict=faf_index_dict,`

As is we get the warning
```
WARNING (gnomad.utils.release 130): Found unexpected frequency metadata groupings: {'gen_anc'}.
```

**Review by jkgoodrich — COMMENTED** [review]
Just 2 small things

File: gnomad_qc/v4/annotations/generate_freq_genomes.py (hunk 2 above)
Comment (jkgoodrich) [reply gnomad_qc/v4/annotations/generate_freq_genomes.py:1009]: nvm, putting it there doesn't work, but maybe add sort_order to the call to `make_freq_index_dict_from_meta`

Comment (jkgoodrich) [reply gnomad_qc/v4/annotations/generate_freq_genomes.py:1009]: or actually it just needs a `hl.literal(` around `faf_meta`

**Review by jkgoodrich — APPROVED** [review]
LGTM!

**Pull request merged**